src/core/server/index.ts

```diff
@@ -121,6 +121,8 @@ export {
   RouteConfigOptionsBody,
   RouteContentType,
   validBodyOutput,
+  ValidateFunctionReturn,
+  ValidateFunction,
```

Comment:
We usually prefix exports from a given core submodule to avoid collision, I think we want to do this here? `RouteXXX` probably? Also I think `RouteValidationFunction` maybe is a better name than `RouteValidateFunction`?